src/js/table.js

```diff
@@ -0,0 +1,235 @@
+(function (Wood) {
+    Wood.Table = Marionette.ItemView.extend({
```

Comment:
I'm not sure we want the table here actually, at least not the DataTable one.  That'll be more specific to ORB and orb.js